build.assets/docker/base.dockerfile

```diff
@@ -55,6 +55,36 @@ RUN export DEBIAN_FRONTEND=noninteractive && set -ex && \
         && apt-get -y autoclean && apt-get -y clean && apt-get autoremove \
         && rm -rf /var/lib/apt/lists/*;
 
+# We need to use a newer version of iptables than debian has available
+# not ideal, but it's easier to run `make install` if we run this inline instead of a multi-stage build
+RUN export DEBIAN_FRONTEND=noninteractive && set -ex \
+        && apt-get update \
+        && apt-get install -q -y --allow-downgrades --no-install-recommends \
+        git \
+        autoconf \
+        libtool \
+        automake \
+        pkg-config \
+        libmnl-dev \
+        make \
+        && mkdir /tmp/iptables.build \
```

Comment:
nit: would it make sense to at least move this to a dedicated RUN step to keep it clearer?

Reply:
I went with this approach to try and limit the image size growth. While we may currently only take the resulting rootfs for packaging with gravity, I don't know if that will always be the case, and we do things like upload to quay.io for use in custom system containers. And I don't want to ship the additional tools in the resulting rootfs if we don't need them.